### DISCUSSION TITLE
Fonts folder misplaced?

### DISCUSSION THREAD

**Discussion by joelrb** [vercel/next.js#69848] [App Router]
In the latest version of NextJS, isn't the fonts folder out of place? 

I know I can drag it to another folder. It's just weird that on project creation it's inside the app folder and in-between my folders.

![Screenshot 2024-09-08 231123](https://github.com/user-attachments/assets/347b3491-15e1-4c00-981d-c87cb729ea39)

**Comment by romhenri** [comment]
In the latest Next versions, fonts folder is placed inside the app dir by default by a design choice, not problem. 

As you said, if you prefer, you can move it to a more intuitive path, like public/fonts, and update your references in CSS to new path.

**Comment by gyanendrasng** [comment]
@romhenri Yes, but what was the reason it was placed in app directory, is there any specific reason?
